### DISCUSSION TITLE
PyCaret in Nature Journal

### DISCUSSION THREAD

**Discussion by DanieleBaranzini** [pycaret/pycaret#2409] [Show and tell]
@moezali1 @ngupta23 

I think it was relevant to tell you that I just published on Nature journal (Impact Factor 9!) a paper where I heavily used PyCaret. In the paper (last page) I referenced this PyCaret comunity: If Nature thinks ML then PyCaret replies : )

at the end I tuned a Quadratic Discriminant Analysis with a curated Feature Engineering via the setup function...

happy reading if you fancy to explore the medical domain and AI services...

[https://www.linkedin.com/in/danielebaranzini/recent-activity/shares/](url)

Daniele Baranzini (Ergonomica)

**Comment by ngupta23** [comment]
This is amazing @DanieleBaranzini. I can not open the link but I am assuming you are [referring to this](https://www.nature.com/articles/s41531-022-00304-z.epdf?sharing_token=vk-4svr2aMDESy5eKwEoN9RgN0jAjWel9jnR3ZoTv0NruIh848F6Qd1SHRKNcfgBGUjvqfjZoGQ9tYyTRBECNhHiiEKy8CG7RCpAibOIxBmLaBEelmECmSThmNdNDKEinX3izRPCbuvtS6hzSEfzIOxP46PCBLDf__Zl69fkYufpiDM6sHP7gvyDT1kdhuSt) paper.

I know it is a dream of a lot of researchers to publish in Nature, so many congratulations for that.

**Reply by DanieleBaranzini** [reply]
@ngupta23 ...indeed that's the paper : )  ...it was a long journey with multiple reviews and it is clear that the old school is not blind to such ML or DL moves  that we are proposing.   

Also PyCaret proved to be reliable effcient and robust....it was a complete success.

Daniele

**Comment by debdarsan** [comment]
@[DanieleBaranzini](https://github.com/DanieleBaranzini)

That is indeed great news for the academicians to get a "Nature" stamp on PyCaret! Congrats!
I have included the reference to your paper in the manuscript I am preparing at this moment.

Dr Debdarsan Niyogi

**Reply by DanieleBaranzini** [reply]
Thanks Dr Debdarsan! appreciated. 
Daniele

**Comment by Rubens1** [comment]
Congratulations for your work
